src/components/sidebar/SidebarGroup.js

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import React, {Component} from 'react';
-import {bem} from '../../lib';
-
+import {bem} from 'lib';
+import ChevronRightIcon from 'mdi-react/ChevronRightIcon';
 import SidebarItem from './SidebarItem';
```

Comment:
it's AngleDownIcon in plotly-icons

Reply:
Cool, I'll make sure to update all of the icons throughout with our `plotly-icons` and add any I use throughout :)